src/components/PressSection/index.tsx

```diff
@@ -40,7 +40,7 @@ export const PressSection: React.FunctionComponent<IPressSection> = ({ id, title
   if (!articles || !articles.length) return null;
 
   return (
-    <Section id={id}>
+    <Section id={id} noPaddingB>
```

Comment:
Same as above, suggest passing this in as a prop and destructuring it like we're doing in the other components

```suggestion
    <Section id={id} {...sectionProps}>
```